packages/syft/src/syft/core/node/common/node_service/role_manager/role_manager_service.py

```diff
@@ -197,6 +274,7 @@ def process(
         msg: INPUT_MESSAGES,
         verify_key: VerifyKey,
     ) -> OUTPUT_MESSAGES:
+        print(RoleManagerService.msg_handler_map[type(msg)])
```

Comment:
NITPICK: Should the print statement be removed?

Reply:
Yep, great catch! 😄